FrontEnd/athena/src/App.js

```diff
@@ -1,84 +1,91 @@
-import React from 'react';
-import { useState, useEffect } from 'react';
+import React from "react";
+import { useState, useEffect } from "react";
+import { Responsive, WidthProvider } from "react-grid-layout";
 
-import logo from './logo.svg';
-import './App.css';
+import "react-grid-layout/css/styles.css";
+import "react-resizable/css/styles.css";
+import logo from "./logo.svg";
+import "./App.css";
 
-import UploadZone from './components/UploadZone';
-import Progress from './components/Progress';
-import EmotionTextChart from './components/EmotionTextChart';
-import EmotionFaceChart from './components/EmotionFaceChart';
-import Transcript from './components/Transcript';
+import UploadZone from "./components/UploadZone";
+import Progress from "./components/Progress";
+import EmotionTextChart from "./components/EmotionTextChart";
+import EmotionFaceChart from "./components/EmotionFaceChart";
+import Transcript from "./components/Transcript";
 
 // import emotionFaceData from './fakeData/faceEmotion';
 
+import Transcription from "./components/Transcription";
 
+const ResponsiveGridLayout = WidthProvider(Responsive);
 
 function App() {
-
   const [showProgres, setShowProgress] = useState(false);
   const [progress, setProgress] = useState(0);
 
   const [emotionFaceData, setEmotionFaceData] = useState([]);
   const [emotionTextData, setEmotionTextData] = useState(null);
   const [transcript, setTranscript] = useState(null);
 
-
+  const transcription = [
+    {
+      time: "2:00",
+      text: "hello",
+      person: "Mark",
+    },
+    {
+      time: "2:01",
+      text: "hello",
+      person: "me",
```

Comment:
if it's "me" then it's the current therapist. when it's not "me", we render the text as blue/purple color.